README.md

```diff
@@ -1,29 +1,66 @@
-# goe-cli
+# GoE — Git on Ethereum
 
-**goe-cli** is a decentralized Git solution designed to manage repositories on-chain. It provides two main components:
+GoE is a decentralized Git protocol built on Ethereum and EthStorage, providing a new standard for trustworthy, on-chain code hosting.  
+It redefines code ownership and verifiability, making your repositories free, secure, and permanently accessible, like Bitcoin.
 
-1. **Wallet CLI** – manage your blockchain wallets.
-2. **Repo CLI** – manage repositories and permissions on-chain.
-3. **Git Helper** – integrates with standard Git commands using the `goe://` protocol.
+
+With GoE, your code is:
+- **Censorship-resistant** — leveraging Ethereum’s permissionless network and global consensus for truly open collaboration.
+- **Immutable and traceable** — all commits and history are stored on-chain, verifiable for the long term.
+- **Web3-native** — fully compatible with Ethereum wallets, DAOs, DApps, and identity systems.
+- **Fully usable** — no new blockchain or extra node infrastructure required.
+- **Git-native** — seamless compatibility with standard Git commands via the `goe://` protocol.
 
 ---
 
-## 1. Wallet CLI
+## How GoE Works
 
-The wallet CLI allows you to create, list, and manage wallets. All wallet-based operations require unlocking the wallet first.
+GoE uses a three-layer model for seamless Git integration:
 
-### Commands
+1. **Git Remote Helper** — handles the `goe://` protocol for all Git commands.
+2. **Ethereum Smart Contracts** — manage branches, commits, and access permissions on-chain.
+3. **EthStorage (EIP-4844 Blob)** — stores large Git packfiles efficiently on Ethereum L2.
+
```

Comment:
We can add our design doc [link](https://github.com/ethstorage/ethfs-git/pull/1) here in case readers want to learn more about the design details

Reply:
fix